pages/faq.vue

```diff
@@ -21,7 +21,7 @@
           </a>
         <v-row id="hero">
           
-          <v-col class="hero-copy" :md="7" :sm="10">
+          <v-col cols="12" :sm="8" :xs="12" class="hero-copy">
```

Comment:
a thing I have recently learned is that "cols" is what is applied at xs, so you don't need to specify that

Reply:
Yeah, I recently learned this also (h/t Colby).  Not sure why I left it like this but can update to only have `:xs="12`